stacker/tests/actions/test_diff.py

```diff
@@ -81,3 +84,61 @@ def test_diff_parameters_no_changes(self):
 
         param_diffs = diff_parameters(old_params, new_params)
         self.assertEquals(param_diffs, [])
+
+
+class TestDiffAction(unittest.TestCase):
+    """Test diff Action class."""
+
+    def test_normalize_json(self):
+        """Ensure _normalize_json parses yaml correctly."""
+        diff = Action(
+            context=Context(config=Config({"namespace": "namespace"}))
+        )
+        with open(os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))),  # noqa
+                               'fixtures',
+                               'cfn_template.yaml'), 'r') as yamlfile:
+            template = yamlfile.read()
+        normalized_template = [
+            '{\n',
+            '    "AWSTemplateFormatVersion": "2010-09-09", \n',
+            '    "Description": "TestTemplate", \n',
+            '    "Outputs": {\n',
+            '        "DummyId": {\n',
+            '            "Value": "dummy-1234"\n',
+            '        }\n',
+            '    }, \n',
+            '    "Parameters": {\n',
+            '        "Param1": {\n',
+            '            "Type": "String"\n',
+            '        }, \n',
+            '        "Param2": {\n',
+            '            "Default": "default", \n',
+            '            "Type": "CommaDelimitedList"\n',
+            '        }\n',
+            '    }, \n',
+            '    "Resources": {\n',
+            '        "Bucket": {\n',
+            '            "Properties": {\n',
+            '                "BucketName": {\n',
+            '                    "Fn::Join": [\n',
+            '                        "-", \n',
+            '                        [\n',
+            '                            {\n',
+            '                                "Ref": "AWS::StackName"\n',
+            '                            }, \n',
+            '                            {\n',
+            '                                "Ref": "AWS::Region"\n',
+            '                            }\n',
+            '                        ]\n',
+            '                    ]\n',
+            '                }\n',
+            '            }, \n',
+            '            "Type": "AWS::S3::Bucket"\n',
+            '        }, \n',
+            '        "Dummy": {\n',
+            '            "Type": "AWS::CloudFormation::WaitConditionHandle"\n',
+            '        }\n',
+            '    }\n',
+            '}\n'
+        ]
+        self.assertEquals(normalized_template, diff._normalize_json(template))
```

Comment:
We should just pull this off of the `Diff` class and into a function. Easier to test since you don't need to initialize anything.